lib/editor/mxEditor.d.ts

```diff
@@ -1266,7 +1266,7 @@ declare module 'mxgraph' {
      * @param source
      * @param target
      */
-    createEdge(source: any, target: any): void;
+    createEdge(source: any, target: any): any;
```

Comment:
Hi @BlankFX, thanks for your pr! 
I checked the source code, for better type declaration, I recommend using mxCell instead of any.
```js
/**
 * Function: createEdge
 *
 * Uses <defaultEdge> as the prototype for creating new edges
 * in the connection handler of the graph. The style of the
 * edge will be overridden with the value returned by
 * <getEdgeStyle>.
 */
mxEditor.prototype.createEdge = function (source, target)
{
	// Clones the defaultedge prototype
	var e = null;

	if (this.defaultEdge != null)
	{
		var model = this.graph.getModel();
		e = model.cloneCell(this.defaultEdge);
	}
	else
	{
		e = new mxCell('');
		e.setEdge(true);

		var geo = new mxGeometry();
		geo.relative = true;
		e.setGeometry(geo);
	}

	// Overrides the edge style
	var style = this.getEdgeStyle();

	if (style != null)
	{
		e.setStyle(style);
	}

	return e;
};
```

```suggestion
    createEdge(source: any, target: any): mxCell;
```